src/adapter/src/repositories/grpc/models/gpt_answer.rs

```diff
@@ -0,0 +1,40 @@
+use rust_core::common::errors::CoreError;
+use tonic::transport::Channel;
+
+use grpc_interface::interfaces::gpt_answer::gpt_answer::{
+    gpt_answer_service_client::GptAnswerServiceClient, GetAnswerPayload,
+};
+
+pub struct GptAnswerGrpcClient {
+    client: GptAnswerServiceClient<Channel>,
+}
+
+impl GptAnswerGrpcClient {
+    fn new(channel: Channel) -> Self {
+        let client = GptAnswerServiceClient::new(channel);
+        Self { client }
+    }
+
+    pub async fn get_instance(uri: &'static str) -> Result<Self, CoreError> {
```

Comment:
`get_instance` is something used for getting an existed object, not for creating a new object